ui/packages/consul-ui/app/models/auth-method.js

```diff
@@ -22,4 +24,10 @@ export default class AuthMethod extends Model {
   @attr('number') ModifyIndex;
   @attr() Datacenters; // string[]
   @attr() meta; // {}
+
+  @computed('MaxTokenTTL')
+  get TTL() {
```

Comment:
Could we use a name that still references the fact that its the TTL of the token that is created by the auth method that we are talking about here, not the auth-method?

Reply:
renamed to be `TokenTTL`